packages/zowe-explorer-api/src/extend/MainframeInteraction.ts

```diff
@@ -500,6 +500,16 @@ export namespace MainframeInteraction {
          */
         issueTsoCommandWithParms?(command: string, parms?: zostso.IStartTsoParms): Promise<zostso.IIssueResponse>;
 
+        /**
+         * Issues a TSO Command without the need for account information
+         *
+         * @param {string} command
+         * @param {zostso.IStartTsoParms} parms
+         * @returns {Promise<zostso.IIssueResponse>}
+         * @memberof ICommand
+         */
+        issueTsoCmdWithParms?(command: string, parms?: zostso.IStartTsoParms): Promise<zostso.IIssueResponse>;
+
```

Comment:
Sorry for not addressing the question sooner.
This is just to avoid migrating extenders to use the new z/OSMF APIs introduced in z/OS 2.4

I just added a deprecation note to the `...Command...` one, that may get people moving towards the new API by V4 🙏